framework/common/src/main/java/edu/kit/kastel/mcse/ardoco/core/api/diagramrecognition/Box.java

```diff
@@ -69,7 +126,7 @@ public int area() {
      * @return a UUID of the box
      */
     public String getUUID() {
-        return uuid;
+        return getName();
```

Comment:
Box now extends DiagramElement which in turn extends entity. In my understanding entity names are unique and thus the function has now kind of become obsolete (Since we also set the name to be the UUID, they are definitely in this case). The function should probably be removed entirely and getName can be used instead